private/headsurface.m

```diff
@@ -195,6 +195,8 @@
     otherwise
       ft_error('other surfaces cannot be constructed this way');
   end
+else
+    ft_error('Headmodel or headshape required.')
```

Comment:
@robertoostenveld This might be too low level? I tested the cases I could think of and also checked where `headsurface.m` is used throughout the code base, but cannot ensure there is not a case where a resulting empty `pos` might be wanted.

The background is that not providing a headmodel but specifying `cfg.moveinward` results in the error
``` Undefined function or variable 'pos'.```
which is maybe not so informative for novice users, what went wrong. Today I saw that an empty `cfg.headmodel` throws errors at other positions for surface models as well, thus the lower level "catch" of the problem.

Reply:
it is indeed appropriate to check that one of the if-elseif-elseif clauses is actually being used